secure-boot-man.md

```diff
@@ -0,0 +1,121 @@
+SECURE-BOOT 8 "December 2020" "secure-boot 1.5.0" "User Manual"
+==================================================
+
+# NAME
+  secure-boot - UEFI SecureBoot on ArchLinux
+
+# SYNOPSIS
+  secure-boot [OPTIONS] \<command\>
+
+# DESCRIPTION
+`secure-boot` is a userspace application to use UEFI secure boot on ArchLinux.
+It generates keys to sign the bootimage, enrolls the keys into the UEFI
+firmware (experimental), signs the bootimage and creates EFI boot entries (no
+intermediate bootloader is needed).
+
+This process is fully configurable by putting the options into
+`/etc/secure-boot/config.mk` (in makefile format) or specify them on the
+command line (see `EXAMPLES`).
```

Comment:
I guess `EXAMPLES` should be `OPTIONS`, right?

Reply:
No, I want to link to the examples where some options are passed on the command line.

Reply:
ah ok, below is `EXAMPLE` not `EXAMPLES`

Reply:
Oh, `man man` says the convention is `EXAMPLE` but they use `EXAMPLES` later. I will use `EXAMPLES` too.